pkg/kubelet/prober/prober.go

```diff
@@ -129,6 +135,9 @@ func (pb *prober) probe(probeType probeType, pod *v1.Pod, status v1.PodStatus, c
 	} else {
 		klog.V(3).InfoS("Probe succeeded", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
 	}
+	if !klog.V(3).Enabled() {
```

Comment:
This will print a trace on every log level except V=3. I don't think we want to print traces for any log level lower than V=5.

Reply:
If log level is equals or higher than V=3, the original code would print Probe successful logs, so we don't need this trace at all,  however, if log level is lower than 3, we can also use the trace to print the logs not too verbose without changing the level, that is every 15 mimutes or longer, this is the intention for the issue which opened by @SergeyKanzhelev.

If this work has no value, we can stop.